corehq/ex-submodules/couchforms/management/commands/purge_forms_and_cases.py

```diff
@@ -0,0 +1,132 @@
+import csv
+
+from django.core.management.base import BaseCommand, CommandError
+from corehq.apps.app_manager.models import Application
+from casexml.apps.case.xform import get_case_ids_from_form
+from corehq.form_processor.interfaces.dbaccessors import FormAccessors, CaseAccessors
+from dimagi.utils.django.management import are_you_sure
+
+
+X_FORM_HEADER = ['Version', 'Form Id', 'Domain']
+CASE_HEADER = ['CaseID', 'CaseType', 'CaseOpenedOn', 'CaseName', 'CaseUserId', 'CaseXFormIds']
+X_FORM_FILENAME = 'X-Form-Details.csv'
+CASE_FILE_NAME = 'Case-Details.csv'
+
+
+# Expects 4 arguments in order : domain app_id version_number test_run
+# ex: ./manage.py purge_forms_and_cases testapp c531daeece0633738c9a3676a13e3d4f 88 yes
+# domain is included with app_id to ensure the user knows what app to delete
+# version_number to delete data accumulated by versions BEFORE this : integer
+# test_run should be yes(case-sensitive) for a test_run and anything otherwise
+# though deletion would be re-confirmed so no panic
```

Comment:
you can add this as an attribute on the class called `help` and then it will be be printed out when someone runs `./manage.py purge_forms_and_cases --help`